edb/edgeql/tracer.py

```diff
@@ -406,9 +409,9 @@ def trace_Path(node: qlast.Path, *,
                     if ptr is None:
                         # Invalid pointer reference, bail.
                         return
-
-                    source_name = ptr.source.get_name(ctx.schema)
-                    ctx.refs.add(f'{source_name}@{step.ptr.name}')
+                    else:
```

Comment:
Why this `else`? It does nothing at the cost of reduced readability.

Reply:
Yeah, this is definitely a remnant of some intermediate code I was trying out. Removed.